sunbeam-python/sunbeam/provider/maas/deployment.py

```diff
@@ -288,6 +288,18 @@ def get_space(self, network: Networks) -> str:
             raise ValueError(f"Space for network {network.value} not set.")
         return space
 
+    def get_spaces(self, networks: list[Networks]) -> set:
```

Comment:
```suggestion
    def get_spaces(self, networks: list[Networks]) -> set[str]:
```

Reply:
This code chunk is removed in the subsequent push